libs/net/dal/Migrations/1.0.167/Up/PostUp/00-ChartTemplate.sql

```diff
@@ -0,0 +1,57 @@
+DO $$
+BEGIN
+
+-- Update name field in report template to match all environments:
+UPDATE public."chart_template"
+SET "template" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ChartEngineContentModel>
```

Comment:
Fixed logic to ensure there are the correct number and order of values for topic types.